src/monitor-mint-calls.spec.js

```diff
@@ -0,0 +1,108 @@
+const ethers = require('ethers');
+const { createTransactionEvent } = require('forta-agent');
+
+// override the timeout from the agent configuration JSON file before the handler is imported
+jest.mock('../agent-config.json', () => ({
+  monitorMintCalls: { rpcTimeoutMilliseconds: 1000 },
+}));
+
+const { handleTransaction, createAlert } = require('./monitor-mint-calls');
+
+// load required contract addresses
+const {
+  VotingToken: votingTokenAddress,
+  Voting: votingAddress,
+} = require('../contract-addresses.json');
+
+// get the abi for the VotingToken contract
+const { abi: votingTokenAbi } = require('../abi/VotingToken.json');
```

Comment:
NPM method maybe

Reply:
Added import of `@uma/contracts-node` module and associated function calls.